backend/index.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express')
+const app = express()
+app.use(express.json())
+
+const mongoose = require('mongoose');
+const connection_url = "mongodb+srv://newUser:newPassword@cluster0.loqju.mongodb.net/RecipesDB?retryWrites=true&w=majority";
+
+mongoose.connect(connection_url)
+.then(() => console.log('Successfully connected'))
+.catch((error) => console.error(`Could not connect due to $(error)`));
+
+const RecipeSchema = new mongoose.Schema({
+    "recipeName" : String,
+    "previewDesc" : String, 
+    "recipeDescription" : String, 
+    "recipeImage" : String,
+    "ingredientList" : [String],
+    "steps" : [String]
+}, {collection: 'Recipes'});
+
+const Recipe = mongoose.model('Recipe', RecipeSchema);
+// ---- End of Schema/Model
+
+
+app.get('/', (req, res) => {
+  res.send('Hello world!');
+});
+
+app.get('/api/recipe', async (req, res) => {
```

Comment:
It definitely isn't a requirement, but putting a short description of what each endpoint is a good practice!